validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -117,6 +124,8 @@
   private LinkedBlockingQueue<ProposerSlashing> slashings = new LinkedBlockingQueue<>();
   private HashMap<UnsignedLong, Eth1DataWithIndexAndDeposits> eth1DataCache = new HashMap<>();
 
+  @VisibleForTesting public Map<Integer, Integer> committeeIndexTTL = new HashMap<>();
```

Comment:
I wouldn't make this public. If we really need it to be visible for testing add a `getCommitteeIndexTtl` method which can return an immutable view of the map. Otherwise we're exposing mutable state which the object should be encapsulating.

But at the moment there don't seem to be any tests using it so it could probably just be private.